crates/swc/tests/tsc-references/defaultExportsCannotMerge01.2.minified.js

```diff
@@ -1,24 +1,17 @@
 //// [defaultExportsCannotMerge01.ts]
 //// [m1.ts]
-var Decl, Decl1;
-function Decl() {
+var Decl;
+function Decl1() {
     return 0;
 }
 Object.defineProperty(exports, "__esModule", {
     value: !0
-}), function(target, all) {
-    for(var name in all)Object.defineProperty(target, name, {
-        enumerable: !0,
-        get: all[name]
-    });
-}(exports, {
-    Decl: function() {
-        return Decl;
-    },
-    default: function() {
-        return Decl;
+}), Object.defineProperty(exports, "default", {
```

Comment:
I thougnt this was a bug.
But I found we get the same result of tsc now.

https://www.typescriptlang.org/play?target=1&module=1#code/PTAEAEFsHsBMFcA2BTAXKAxtSMB2ArAZwCgQIAXAQwCcBzZc9AUQGUBWY45ADwAdpq5ULGQAzSkiGj4uDOQCW0XKAAiyDIgAUASlABvYqCOhqDeNWUAGANzEAvpx79BoebnLJq4jMlXrE+obGvACM6LjwkABGnrbGoLwATOGRMdS2Dlx8AkK4lJDIhLyUPn4agfFOOaAAbjSg3KAAvKAhNkFGVS511KAAns2gie0dru6e3r4AkhXGDg5AA